src/DynamoPackages/PackageManagerExtension.cs

```diff
@@ -103,6 +103,7 @@ public void Startup(StartupParams startupParams)
                 new MutatingFileSystem(),
                 new CustomNodePathRemapper(startupParams.CustomNodeManager, DynamoModel.IsTestMode));
 
+            PackageUploadBuilder.SetEngineVersion(startupParams.DynamoVersion);
             var uploadBuilder = new PackageUploadBuilder(dirBuilder, new MutatingFileCompressor());
```

Comment:
Let's pass the version as an argument. `static` state makes it very hard to decouple components for testing.